metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1579,6 +1579,7 @@ class MetalsLanguageServer(
           }
       }
       workspaceSymbols.didChange(source, symbols)
+      symbolDocs.didChange(symbols.map(_.symbol))
```

Comment:
Could we invoke `indexSymbolDefinition` inside the foreach above? I think we have all the needed information and skip the additional steps.

Reply:
Ok, I think did change is fine here, but added a comment in Docstrings